src/dynarec/la64/dynarec_la64_660f.c

```diff
@@ -700,8 +700,11 @@ uintptr_t dynarec64_660F(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t ip, int
             GETGX(v0, 0);
             if (MODREG) {
                 v1 = sse_get_reg_empty(dyn, ninst, x1, (nextop & 7) + (rex.b << 3));
-                VXOR_V(v1, v1, v1);
-                VEXTRINS_D(v1, v0, 0);
+                q0 = fpu_get_scratch(dyn);
```

Comment:
if ex==gx, than v0==v1, and the operationis a NOP...

Reply:
not really, the upper bits still need to be cleared.

Reply:
anyway, if v0 != v1, 1 instruction can be saved, I pushed the optimization.